recipes/parsnp/meta.yaml

```diff
@@ -14,21 +14,18 @@ source:
     sha256: '{{sha256osx}}'                                           # [osx]
 
 build:
-  number: 4
-  skip: True  # [py2k]
+  number: 0
 
 requirements:
   build:
   host:
-    - zlib
-    - llvm-openmp  # [osx]
-    - openmp
   run:
     - llvm-openmp  # [osx]
     - openmp
     - harvesttools
     - fasttree
     - phipack
+    - python >=3
```

Comment:
I made this `>=3` because of the previous `skip: True  # [py2k]` line.
Is that correct or is this also meant to be compatible with Python 2.7?

Reply:
Nope. Python 3+ is required. I had originally had the `- python >= 3`, however one of the Bioconda members changed it to the `skip: True # [py2k]`. Not sure why.

Reply:
Yeah, the `conda-build` behavior with when to use `skip ...py...` and when to use `python >=...` is a little finicky and (rightfully) confuses many..

Alright! Then we're good to go on this one!